external/llvm-project/mlir/test/CMakeLists.txt

```diff
@@ -10,6 +10,14 @@ endif()
 set(MLIR_CMAKE_DIR
   "${CMAKE_BINARY_DIR}/lib${LLVM_LIBDIR_SUFFIX}/cmake/mlir")
 
+if(WIN32)
```

Comment:
These are patches to upstream - are you willing to take responsibility for upstreaming them?

Furthermore, changes to `external/llvm-project/` should be separate from other changes for git subtree reasons